lib/src/voip/backend/mesh_backend.dart

```diff
@@ -658,7 +666,29 @@ class MeshBackend extends CallBackend {
     return;
   }
 
-  Future<void> _onIncomingCall(
+  void _onIncomingCallSetup(GroupCallSession groupCall, CallSession newCall) {
+    if (newCall.room.id != groupCall.room.id ||
+        newCall.state != CallState.kRinging ||
```

Comment:
Can you look at _onIncomingCallStart and add comments to all the conditionals and separate them out? It's easier to debug why calls are breaking then. Also make sure to namespace the comments so we know what function has printed them.

Reply:
you mean logs and not comments right?

Reply:
if it's logs, then done!